XmlFormatter/src/Hotfolder/HotfolderContainer.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using XmlFormatter.src.Enums;
+using XmlFormatter.src.Interfaces.Formatter;
+using XmlFormatter.src.Interfaces.Hotfolder;
+
+namespace XmlFormatter.src.Hotfolder
+{
+    class HotfolderContainer : IHotfolder
+    {
+        /// <summary>
+        /// The mode to use for formatting
+        /// </summary>
+        private ModesEnum mode;
+
+        /// <inheritdoc/>
+        public ModesEnum Mode
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this an auto-implemented property and remove its backing field.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5053278)